### ISSUE TITLE
AttributeError raised when performing NN cross correlation with multiple datasets

### ISSUE THREAD

**Issue by marxide** [rmjarvis/TreeCorr#37]
I received the following error when attempting to run corr2:
`AttributeError: Cannot provide both file_name2 and rand_file_list2.`

The following is a snippet from the configuration file I was using
```
file_name = dataset1.dat
file_name2 = dataset2.dat
rand_file_list = random_1_list.txt
rand_file_list2 = random_2_list.txt
```
where `rand_file_list` and `rand_file_list2` contain a list of 10 files each.

I was able to (apparently!) fix the problem by changing https://github.com/rmjarvis/TreeCorr/blob/fb44b460054b8b8d7a0da05b3f4210febc6edb77/treecorr/corr2.py#L134 to `cat2 = treecorr.read_catalogs(config, 'file_name2', 'file_list2', 1, logger)`.

**Comment by rmjarvis** [comment]
Yes.  Quite right.  That's exactly the bug.  Sorry about that!  I'll fix it straight away.

**Comment by rmjarvis** [comment]
This is now fixed in version 3.2.2.  Thanks very much for letting me know about this bug.
